docs/apps/transfer/messages.md

```diff
@@ -0,0 +1,37 @@
+<!--
```

Comment:
I think `release/v2.5.x` doesn't have the transfer folder in the docs, so now the backport is trying to add this file only... Maybe we could backport the [PR](https://github.com/cosmos/ibc-go/pull/1554/files) that moved the transfer docs from the spec folder; or we could just remove this file, and update instead [the file that is still in the spec folder in this release branch](https://github.com/cosmos/ibc-go/blob/release/v2.5.x/modules/apps/transfer/spec/04_messages.md).

I guess we will have a similar problem for some of the other back ports for the metadata field...

Reply:
I'll go for the latter option.